dbt/include/bigquery/macros/adapters.sql

```diff
@@ -73,6 +73,12 @@
     TODO: Deep dive into spark sessions to see if we can reuse a single session for an entire
     dbt invocation.
      --#}
+
+    {#-- when a user wants to change the schema of an existing relation, they must intentionally drop the table in the dataset --#}
```

Comment:
Do we specifically only want to do this when the old relation is a table? Or do we always want to drop the old relation if the names collide (e.g. a view is now a table)? If it's the latter, I think this can be boiled down to `{{ drop_relation_if_exists(relation) }}`

Reply:
Oh interesting. Ok, makes sense. Thanks for the context.